vms/platformvm/state/camino.go

```diff
@@ -418,45 +437,42 @@ func (cs *caminoState) Load() error {
 	}
 	cs.lockModeBondDeposit = mode
 
-	return cs.loadDepositOffers()
+	errs := wrappers.Errs{}
+	errs.Add(
+		cs.loadDepositOffers(),
+		cs.loadDeferredValidators(s),
+	)
+	return errs.Err
 }
 
 func (cs *caminoState) Write() error {
+	errs := wrappers.Errs{}
+
 	// Write the singletons (only once after sync)
 	if cs.genesisSynced {
-		if err := database.PutBool(cs.caminoDB, nodeSignatureKey, cs.verifyNodeSignature); err != nil {
-			return fmt.Errorf("failed to write verifyNodeSignature: %w", err)
-		}
-		if err := database.PutBool(cs.caminoDB, depositBondModeKey, cs.lockModeBondDeposit); err != nil {
-			return fmt.Errorf("failed to write lockModeBondDeposit: %w", err)
-		}
+		errs.Add(
+			database.PutBool(cs.caminoDB, nodeSignatureKey, cs.verifyNodeSignature),
+			database.PutBool(cs.caminoDB, depositBondModeKey, cs.lockModeBondDeposit),
+		)
 	}
 
-	if err := cs.writeAddressStates(); err != nil {
-		return err
-	}
-	if err := cs.writeDepositOffers(); err != nil {
-		return err
-	}
-	if err := cs.writeDeposits(); err != nil {
-		return err
-	}
-	if err := cs.writeMultisigOwners(); err != nil {
-		return err
-	}
-	if err := cs.writeNodeConsortiumMembers(); err != nil {
-		return err
-	}
-	if err := cs.writeValidatorRewards(); err != nil {
-		return err
-	}
+	errs.Add(
+		cs.writeDeferredStakers(),
+		cs.writeAddressStates(),
+		cs.writeDepositOffers(),
+		cs.writeDeposits(),
+		cs.writeMultisigOwners(),
+		cs.writeNodeConsortiumMembers(),
+		cs.writeValidatorRewards(),
+	)
 
-	return nil
+	return errs.Err
 }
 
 func (cs *caminoState) Close() error {
 	errs := wrappers.Errs{}
 	errs.Add(
+		cs.deferredValidatorsDB.Close(),
```

Comment:
I suggest to move it below caminoDB, cause its main db and should go first (or last)